cli/tests/init_service.sh

```
@@ -5,11 +5,11 @@ fi
 mkdir -p output/init-service
 cd output/init-service
 
-RUST_BACKTRACE=1 cargo run --release init application service-test-node-application -p . -d postgresql -f prettier -l eslint -v zod -F express -r node -t vitest -m billing-base -m iam-base -D "Test service" -A "Rohin Bhargava" -L 'AGPL-3.0'
-RUST_BACKTRACE=1 cargo run --release init service svc-test -d postgresql -p service-test-node-application -D "Test service"
-RUST_BACKTRACE=1 cargo run --release init service svc-test-mongodb -d mongodb -p service-test-node-application -D "Test service"
+RUST_BACKTRACE=1 cargo run --release init application service-test-node-application -p ./service-test-node-application -o source -d postgresql -f prettier -l eslint -v zod -F express -r node -t vitest -m billing-base -m iam-base -D "Test service" -A "Rohin Bhargava" -L 'AGPL-3.0'
+RUST_BACKTRACE=1 cargo run --release init service svc-test -d postgresql -p service-test-node-application/src/modules -D "Test service"
+RUST_BACKTRACE=1 cargo run --release init service svc-test-mongodb -d mongodb -p service-test-node-application/src/modules -D "Test service"
 
-cd service-test-node-application
+cd service-test-node-application/src/modules
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)